packages/protocol/contracts-0.8/common/GasPriceMinimum.sol

```diff
@@ -174,14 +173,15 @@ contract GasPriceMinimum is
    * When caled for 0x0 or Celo address, it returns gasPriceMinimum().
    * For other addresses it returns gasPriceMinimum() mutiplied by 
    * the SortedOracles median of the token. It does not check tokenAddress is a valid fee currency.
-   * this function will never returns values less than ABSOLUTE_MINIMAL_GAS_PRICE.
-   * If Oracle rate doesn't exist, it returns ABSOLUTE_MINIMAL_GAS_PRICE.
    * @dev This functions assumes one unit of token has 18 digits.
    * @param tokenAddress The currency the gas price should be in (defaults to Celo).
    * @return current gas price minimum in the requested currency
    */
   function getGasPriceMinimum(address tokenAddress) external view returns (uint256) {
-    return Math.max(_getGasPriceMinimum(tokenAddress), ABSOLUTE_MINIMAL_GAS_PRICE);
+    uint256 _gasPriceMinimum = _getGasPriceMinimum(tokenAddress);
+    require(_gasPriceMinimum > 0, "gas price minimum must be greater than zero");
+
+    return _gasPriceMinimum;
```

Comment:
I do think it's a better property that the protocol always gives an offer for this.
When this number is zero, it means that the number should be smaller than 1, but it gets rounded back to zero. So enforcing a 1 here makes the fee more expensive than it would if this had more precision.

It is rounding up to the side of the protocol and it gives the user the option of wether to pay this or not.